README.md

```diff
@@ -161,6 +161,67 @@ If you want to use the deploy configuration of directory from which you are exec
 
 Note: we should NOT manually adapt code in /var/www/vhosts/mf-geoadmin3 directory
 
+
+## Building the project to int and prod (S3)
+
+Deploying a project to int or prod consists of three steps:
+
+- Building
+- Uploading to S3
+- Activating a version, as only one version may be active at time (`index.html`)
```

Comment:
Ok, so only one version at a time is active for dev, int and prod. Now how do we handle branches? If Varnish doesn't point to EC2 instances anymore but directly to S3 bucket, it seems to me that all our branches will be obsolete. I think we need a mechanism to upload branches as well. Or I missed something in the code, in which case can you show me how this can be done in the code.